exercises/error-handling/error_handling_test.py

```diff
@@ -45,7 +53,12 @@ def test_filelike_objects_are_closed_on_exception(self):
         filelike_object.open()
```

Comment:
I don't think that we should be passing an open object to the user's function. It seems strange to me that we'd pass something open to a function that (if we weren't just throwing an exception) may or may not close it - I would expect closing to be handled in the place that it's opened. I think it should be up to the user to open and close the filelike object however they see fit.